app/src/main/java/com/kabouzeid/gramophone/provider/HistoryStore.java

```diff
@@ -136,10 +138,16 @@ public boolean contains(long id) {
         return containsId;
     }
 
-    public Cursor queryRecentIds() {
+    public Cursor queryRecentIds(@NonNull Context context) {
+        long cutoff = PreferenceUtil.getInstance(context).getRecentlyPlayedCutoffTimeMillis();
```

Comment:
I think it'd be a bit cleaner if the caller passed the `cutoff` value in instead of passing in a `Context`.

Reply:
See efd340c